src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/SymbolKey/SymbolKey.SymbolKeyWriter.cs

```diff
@@ -48,6 +48,7 @@ private enum SymbolKeyType
             Reference = '#',
             Null = '!',
             TypeParameterOrdinal = '@',
+            Preprocessing = '*',
```

Comment:
these are special guys.  the Preprocessing guy shuold move into the named section above.  You can pick any A-Z taht isn't in use yet.